### DISCUSSION TITLE
vcpkg ignores version overwrite

### DISCUSSION THREAD

**Discussion by Thomas1664** [microsoft/vcpkg#22928] [Q&A] [answered]
While trying to update openssl to 3.0.1 I noticed that some packages are incompatible to the new version.
However, vcpkg ignores when I overwrite the version in the manifest file:

```json
 "overrides": [
    {
      "name": "openssl",
      "version-string": "1.1.1m"
    }
  ]
```

Vcpkg still uses the new version, even after I deleted everything inside `buildtrees`, `downloads`, `installed` and `packages`.

**Comment by dg0yt** [comment, marked as answer]
This type of versioning is only supported in user projects (manifest mode), not in portfile manifest. That's the reason why some community members do not like the idea of putting OpenSSL 3 into port openssl.

**Reply by dg0yt** [reply]
This was also answered in https://github.com/microsoft/vcpkg/pull/22878#issuecomment-1029600821.
